src/main/java/spoon/reflect/visitor/DefaultJavaPrettyPrinter.java

```diff
@@ -537,7 +537,7 @@ public <T> void visitCtBinaryOperator(CtBinaryOperator<T> operator) {
 		printer.writeSpace();
 		try (Writable _context = context.modify()) {
 			if (operator.getKind() == BinaryOperatorKind.INSTANCEOF) {
-				_context.forceWildcardGenerics(true);
+				_context.forceWildcardGenerics(env.getComplianceLevel() < 16);
```

Comment:
Can you please move this in a method containing an explanation why we have this condition?